src/containers/Tabs/Admin.js

```diff
@@ -46,12 +47,27 @@ const AdminTabContainer = ({ isOpen }) => {
                 toggleModal("add")();
                 addTransaction(value, PENDING_ADDITION);
             })
-            .on("receipt", () => {
-                openToast(
-                    value,
-                    SUCCESS,
-                    `New admin account processed: ${value}`
+            .on("receipt", receipt => {
+                // Web3js returns true if true, null if false
+                const added = Boolean(
```

Comment:
Again check that the event was emitted and treat the non emission as a further error